### PR TITLE
feat: gnosis blockscout urls

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box 

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes N/A

## Risk

Asset / addresses / explorer links may be borked, test accordingly for Gnosis

## Testing

- See risks 

### Engineering

- ☝🏽 

### Operations

- ☝🏽

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5654** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5654?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5654?utm_source=stack-comment).

File: scripts/generateAssetData/baseAssets.ts [hunk 1]
```diff
@@ -204,7 +204,7 @@ export const arbitrumNova: AssetWithNetworkName = {
   icon: 'https://assets.coincap.io/assets/icons/256/eth.png',
   networkIcon:
     'https://assets.coingecko.com/asset_platforms/images/93/large/AN_logomark.png?1695026131',
-  explorer: 'https://nova.arbiscan.io',
-  explorerAddressLink: 'https://nova.arbiscan.io/address/',
-  explorerTxLink: 'https://nova.arbiscan.io/tx/',
+  explorer: 'https://nova-explorer.arbitrum.io',
```
Comment (gomesalexandre) [inline scripts/generateAssetData/baseAssets.ts:207]: Note, Arbitrum One doesn't have a Blockscout explorer - same goes for BSC and Avalanche, see https://docs.blockscout.com/about/projects

Comment (twblack88) [reply scripts/generateAssetData/baseAssets.ts:207]: yeah putting up commit to leave those

Comment (twblack88) [reply scripts/generateAssetData/baseAssets.ts:207]: jesus 13K changes ok

File: packages/chain-adapters/src/evm/EvmBaseAdapter.ts [hunk 2]
```diff
@@ -222,7 +222,7 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
       [KnownChainIds.GnosisMainnet]: {
         name: 'xDAI',
         symbol: 'xDAI',
-        explorer: 'https://gnosisscan.io/',
+        explorer: 'gnosis.blockscout.com',
```
Comment (kaladinlight) [inline packages/chain-adapters/src/evm/EvmBaseAdapter.ts:225]:
```suggestion
        explorer: 'https://gnosis.blockscout.com',
```

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**